docs/developer/concepts.ipynb

```diff
@@ -291,6 +291,84 @@
     "    ok = True\n",
     "assert not ok"
    ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "`coords` always refers to (aligned) coords in dataset, cannot add or erase via item:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "try:\n",
+    "    ds['a'].coords['fail'] = 1.0 * sc.units.m\n",
+    "except RuntimeError:\n",
+    "    ok = False\n",
+    "else:\n",
+    "    ok = True\n",
+    "assert not ok"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "ds.coords['xx'] = 1.0 * sc.units.m\n",
+    "assert 'xx' in ds['a'].coords\n",
+    "try:\n",
+    "    del ds['a'].coords['xx']\n",
+    "except RuntimeError:\n",
+    "    ok = False\n",
+    "else:\n",
+    "    ok = True\n",
+    "assert not ok"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "`meta` contains dataset coords as well as item attributes, cannot add or erase, since ambiguous:"
```

Comment:
minor point, use abbreviated or full, but maybe not mix `coords` and `attributes`